docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -373,6 +373,11 @@ statements, the connector supports the following features:
 - {doc}`/sql/show-create-table`
 - {doc}`/sql/comment`
 
+### ALTER TABLE
```

Comment:
Add a `see also` link to  (same as done for Data management)

https://trino.io/docs/current/connector/delta-lake.html#sql-support 

for `ALTER TABLE` option